README.md

```diff
@@ -1,268 +1,15 @@
-`hardware_interface::StateInterface` to a topic of type `sensor_msgs/msg/JointState`.
-The following is a basic configuration of the controller.
-
-```yaml
-joint_state_controller:
-  ros__parameters:
-    type: joint_state_controller/JointStateController
-```
-
-This controller creates an action called `/cart_pole_controller/follow_joint_trajectory` of type `control_msgs::action::FollowJointTrajectory`.
-
-```yaml
-cart_pole_controller:
-  ros__parameters:
-    type: joint_trajectory_controller/JointTrajectoryController
-    joints:
-       - slider_to_cart
-    write_op_modes:
-       - slider_to_cart
-```
-#### Executing the examples
-
-There are some examples in the `gazebo_ros2_control_demos` package. These examples allow to launch a cart in a 30 meter rail.
-
-![](img/cart.gif)
-
-You can run some of the configuration running the following commands:
-
-```bash
-ros2 launch gazebo_ros2_control_demos cart_example_position.launch.py
-ros2 launch gazebo_ros2_control_demos cart_example_velocity.launch.py
-ros2 launch gazebo_ros2_control_demos cart_example_effort.launch.py
-ros2 launch gazebo_ros2_control_demos diff_drive.launch.py
-ros2 launch gazebo_ros2_control_demos tricycle_drive.launch.py
-```
-
-Send example commands:
-
-When the Gazebo world is launched you can run some of the following commads to move the cart.
-
-```bash
-ros2 run gazebo_ros2_control_demos example_position
-ros2 run gazebo_ros2_control_demos example_velocity
-ros2 run gazebo_ros2_control_demos example_effort
-ros2 run gazebo_ros2_control_demos example_diff_drive
-ros2 run gazebo_ros2_control_demos example_tricycle_drive
-```
-
-The following example shows parallel gripper with mimic joint:
-
-![](img/gripper.gif)
-
-
-```bash
-ros2 launch gazebo_ros2_control_demos gripper_mimic_joint_example.launch.py
-```
-
-Send example commands:
-
-```bash
-ros2 run gazebo_ros2_control_demos example_gripper
-```
-
-#### Gazebo + Moveit2 + ROS 2
-
-This example works with [ROS 2 Foxy](https://index.ros.org/doc/ros2/Installation/Foxy/).
-You should install Moveit2 from sources, the instructions are available in this [link](https://moveit.ros.org/install-moveit2/source/).
-
-The repository with all the required packages are in the [gazebo_ros_demos](https://github.com/ros-controls/gazebo_ros_demos/tree/ahcorde/port/ros2).
-
-```bash
-ros2 launch rrbot_moveit_demo_nodes rrbot_demo.launch.py
-```
-
-![](img/moveit2.gif)
+ROS 2 Distro | Branch | Build status | Documentation
+:----------: | :----: | :----------: | :-----------:
+**Rolling** | [`master`](https://github.com/ros-controls/gazebo_ros2_control/tree/master) | [![Gazebo ros2 control CI](https://github.com/ros-controls/gazebo_ros2_control/actions/workflows/ci.yaml/badge.svg?branch=master)](https://github.com/ros-controls/gazebo_ros2_control/actions/workflows/ci.yaml) | [Documentation](https://control.ros.org/master/index.html) <br /> [API Reference](https://control.ros.org/master/doc/api/index.html)
```

Comment:
Add Iron

Reply:
I added it pointing to the master branch, is this fine?